### PR TITLE
[12.0][ADD] helpdesk_mgmt_ticket_kanban_seq

### PR DESCRIPTION
**Pull request by dalonsod**
Enables sorting tickets across kanban view.

Actually changes ticket order for every view by default (but compatible with e.g. `helpdesk_mgmt_ticket_tree_recent` in this repo), allowing to re-order in kanban view on an certain column, when the priority is the same.

cc @lmiguens-solvos

### CODE REVIEW

File: helpdesk_mgmt_ticket_kanban_seq/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: helpdesk_mgmt_ticket_kanban_seq
+#: model:ir.model.fields,help:helpdesk_mgmt_ticket_kanban_seq.field_helpdesk_ticket__sequence
+msgid "Gives the sequence order when displaying a list of tickets."
+msgstr "Proporciona ordernación por secuencia en la presentación de la lista de tickets"
```
Comment (lmiguens-solvos) [inline helpdesk_mgmt_ticket_kanban_seq/i18n/es.po]: Typo "ordernacion"

**Review by lmiguens-solvos — APPROVED** [review]
LGTM.
In the tests I did the ordering worked as expected: first priority and then sequence

File: helpdesk_mgmt_ticket_kanban_seq/i18n/es.po (hunk 1 above)
Comment (dalonsod) [reply helpdesk_mgmt_ticket_kanban_seq/i18n/es.po]: Fixed

**Comment by dalonsod** [comment]
Good, it's time to merge...

**Pull request merged**